src/main/java/com/crossover/trial/journals/Application.java

```
@@ -0,0 +1,24 @@
+package com.crossover.trial.journals;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.context.annotation.ComponentScan;
+
+@ComponentScan
+@EnableAutoConfiguration
+public class Application {
+
+	public static final String ROOT;
+
+	static {
+		ROOT = System.getProperty("upload-dir", System.getProperty("user.home") + "/upload");
```

Unsafe directory path handling

If "upload-dir" Java properties is set to invalid directory path, this will cause issue later during file upload and medical journal retrieval.

Fail fast during boottime by implementing additional validation logic to verify valid directory path.